tests/regression/lp-1898038/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: docker-support and multipass-support should mount base /etc/apparmor.d
+prepare: |
+    tests.cleanup prepare
+    snap pack test-docker-support-app
+    tests.cleanup defer rm -f test-docker-support-app_1_all.snap
+    snap install --dangerous ./test-docker-support-app_1_all.snap
+    tests.cleanup defer snap remove --purge test-docker-support-app
+    snap pack test-multipass-support-app
+    tests.cleanup defer rm -f test-multipass-support-app_1_all.snap
+    snap install --dangerous ./test-multipass-support-app_1_all.snap
+    tests.cleanup defer snap remove --purge test-multipass-support-app
+execute: |
+    # wait a bit to ensure the mount namespace exists
+    sleep 5
```

Comment:
If you switch the app to a non-service you can run it like this: (assuming you rename the app to `sh` for brevity:
```
test-snapd-multipass-support.sh -c true
# the mount namespace is now ready
```

Reply:
Thanks - I just added this in abf0f62